Source/SIMPLib/Common/SIMPLArray.hpp

```diff
@@ -244,21 +297,39 @@ class IVec2 : public SIMPLArray<T, 2>
   }
 
   /**
-   * @brief toTuple Converts the internal data structure to a std::tuple<T, T, T>
+   * @brief this is a convenience function that helps with the python bindings
+   */
+  inline void setValues(const T& x, const T& y)
+  {
+    (*this)[0] = x;
+    (*this)[1] = y;
+  }
+
+  /**
+   * @brief values Converts the internal data structure to a std::tuple<T, T, T>
    * @return
    */
-  std::tuple<T, T> toTuple() const
+  std::tuple<T, T> values() const
```

Comment:
toTuple was probably the better name.  Asking an array for its values and getting a tuple is not my expected result if I just saw a list of auto-complete names in an IDE.  I would expect a pointer or vector to come back from just this method name.

Reply:
Somewhat agree. I did this change because the Python bindings need a method to return a tuple but inside of Python it is a list which I thought would be confusing inside of python. I wasn't super sure about the name but nothing else really worked. Let me take a look at the Python binding code and see if something can be done to have it both ways.

Reply:
@mmarineBlueQuartz I fixed this in the latest push